app/services/idv/usps_mail.rb

```diff
@@ -16,6 +17,12 @@ def any_mail_sent?
       user_mail_events.any?
     end
 
+    def most_recent_otp_expired?
+      return false unless any_mail_sent?
+
+      user_mail_events.first.updated_at < USPS_CONFIRMATION_WINDOW_DAYS.days.ago
```

Comment:
(Hi, I know this is already merged, but the code is here, so I'm just gonna comment anyways)

I don't think that `updated_at` is a good column to use because a lot of things could update this record. I think we should add a migration for an explicit `sent_at` (or technically `queued_at`) column that indicates when the mail event was sent, so we have a clear value that should expire?